src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,26 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <UserInfo user={post.user} />
```

Comment:
```suggestion
      {post.user && <UserInfo user={post.user} />}
```
technically user can be undefined and better to make condition rendering here.